plugin/RequestVariablesSanitizer.php

```diff
@@ -42,6 +42,20 @@ public function getFilteredGet(string $name, int $filter = FILTER_DEFAULT, $opti
 		return $this->getFiltered($_GET, $name, $filter, $option);
 	}
 
+	/**
+	 * @return array
+	 */
+
+	public function getParameterPage()
+	{
+		$pagedData = [];
+		foreach ($_GET as $key => $value) {
+			if (strpos($key, 'page_of_id_') === 0) {
+				$pagedData[] = $key;
+			}
+		}
+		return $pagedData;
+	}
```

Comment:
Is this function good placed as a public function in RequestVariablesSanitizer? I think we don't need getParameterPage() for other function.
What do you think?
Should we place this code in SearchParametersModelBuilderEstate->addParametersForPagination ?

Reply:
@andernath You are correct, currently getParameterPage() is not needed for other functions. 

I placed this code in SearchParametersModelBuilderEstate->addParametersForPagination. When placing the code at this will make the code more cohesive and easier to maintain.